README.md

```diff
@@ -6,10 +6,29 @@ Respond to short response questions in clear, concise sentences directly within
 ### Short Response Questions
 **1. What is recursion?**
 
+    Recursion is the concept of a function calling upon itself in their function body.
+    This is especially useful if you're function relies on a lot of looping to get your expected output.
+    For recursion to not cause an infinite loop, or a 1000-round loop if your language has a limit on 
+    recursives functions like python does, you must provide a base case, and a recursive case. A base
+    case to control the action taken before the function loops itself and a recursive case to terminate the
+    recursive loop.
+
 **2. What type of problems are best solved using recursion?**
 
+    The type of problems that recursion can be geared towards typically require a decent amount of looping or 
+    iterating. Though, it can be used in multiple scenarios. One of which is searching for an element, or 
+    traversing through a data structures that contain multiple other other pieces of data. hi
```

Comment:
> hi

Hi! :+1: Right on with traversing some data structures. This would be a good place to talk more about problems where you can work towards a base case.